changelog.d/3578.bugfix

```diff
@@ -0,0 +1,14 @@
+Synapse doesn’t allow for media resources to be played directly from
```

Comment:
sorry, this file just needs to be a short statement, since it ends up in the [changelog](https://github.com/matrix-org/synapse/blob/master/CHANGES.md).

Reply:
The statement was shortened